invenio_communities/communities/records/mappings/os-v1/communities/communities-v2.0.0.json

```diff
@@ -0,0 +1,482 @@
+{
+  "settings": {
+    "analysis": {
+      "char_filter": {
+        "strip_special_chars": {
+          "type": "pattern_replace",
+          "pattern": "[\\p{Punct}\\p{S}]",
+          "replacement": ""
+        }
+      },
+      "analyzer": {
+        "accent_edge_analyzer": {
+          "tokenizer": "standard",
+          "type": "custom",
+          "char_filter": ["strip_special_chars"],
+          "filter": [
+            "lowercase",
+            "asciifolding",
+            "edgegrams"
+          ]
+        }
+      },
+      "filter": {
+        "lowercase": {
+          "type": "lowercase",
+          "preserve_original": true
+        },
+        "asciifolding": {
+          "type": "asciifolding",
+          "preserve_original": true
+        },
+        "edgegrams": {
+          "type": "edge_ngram",
+          "min_gram": 2,
+          "max_gram": 20
```

Comment:
did you gather somewhere your findings about the performance changes after applying these?

Reply:
I tested the performance impacts of the new mappings in https://github.com/inveniosoftware/invenio-vocabularies/pull/372/files and subsequently adjusted the PR so that it wouldn't cause a major increase (my changes took the index from 150mb to 175mb). 

I discussed with @slint and as there are only ~20k communities in Zenodo (and probably similar or lesser numbers of communities across Invenio instances) it is unlikely that the resulting index will increase to a significant size